fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/domain/LoanApplicationTerms.java

```diff
@@ -595,13 +596,26 @@ public Money pmtForInstallment(final PaymentPeriodsInOneYearCalculator calculato
         final int periodsElapsed = periodNumber - 1;
         // with periodic interest for default month and year for
         // equal installment
-        final BigDecimal periodicInterestRateForRepaymentPeriod = periodicInterestRateWithOriginalRate(calculator, mc,
-                DaysInMonthType.DAYS_30, DaysInYearType.DAYS_365, periodStartDate, periodEndDate, true);
+        final BigDecimal periodicInterestRateForRepaymentPeriod = periodicInterestRate(calculator, mc, DaysInMonthType.DAYS_30,
+                DaysInYearType.DAYS_365, periodStartDate, periodEndDate, true, false);
         Money totalPmtForThisInstallment = calculateTotalDueForEqualInstallmentRepaymentPeriod(periodicInterestRateForRepaymentPeriod,
```

Comment:
@faheem205 the idea of using the method periodicInterestRateWithOriginalRate() was to maintain a constant EMI for a loan even after a rescheduling